feat: `astro info` command (#3653)
withastro/docs: src/content/docs/en/reference/cli-reference.mdx

            check  Check your project for errors.
              dev  Start the development server.
             docs  Open documentation in your web browser.

>              info  Emit useful information about Astro current setup. Useful for debugging.

Reviewer: ```suggestion
             info  List info about your current Astro setup.
```
This is way longer than the others. Do you think something like this is good enough?

Author: No problem, I will update the code too

Launches the Astro Docs website directly from the terminal.

> ## `astro info`

Author: I thought that the commands were in alphabetic order.

Reviewer: Apparently not :joy:

> Reports useful information about the Astro environment. Useful to providing information when opening an issue.

Reviewer: ``````suggestion
Reports useful information about your current Astro environment. Useful for providing information when opening an issue.

```shell
astro info
```

Example output:

```
Astro version            v2.6.6
Package manager          pnpm
Platform                 darwin
Architecture             arm64
Adapter                  @astrojs/vercel/serverless
Integrations             None
```
``````
Snagged this from the changeset!
